Detections/InfobloxNIOS/ExcessiveNXDOMAINDNSQueries.yaml

```diff
@@ -22,7 +22,7 @@ query: |
   let threshold = 200;
   InfobloxNIOS
   | where TimeGenerated >= timeframe
-  | where LogType =~ "named" and Type =~ "client"
+  | where ProcessName =~ "named" and Log_Type =~ "client"
   | where isnotempty(ResponseCode)
   | where ResponseCode =~ "NXDOMAIN"
   | summarize count() by Client_IP, bin(TimeGenerated,15m)
```

Comment:
Please join the summarized results to original table and bring in other contextual fields in the output. 
Current output does not provide any helpful information to analysts other that showing IP addresses and count. 

Also populate and map entities (IPCustomEntity, HostCustomEntity) at the end of the query.
Check example queries
- https://github.com/Azure/Azure-Sentinel/blob/master/Detections/DnsEvents/DNS_HighReverseDNSCount_detection.yaml
- https://github.com/Azure/Azure-Sentinel/blob/master/Detections/DnsEvents/DNS_HighNXDomainCount_detection.yaml

Reply:
join has been added